### DISCUSSION TITLE
assembler do Adding new instruction

### DISCUSSION THREAD

**Discussion by Peter-Jacob** [adesutherland/CREXX#88] [Q&A]
C:\Users\PeterJ\CLionProjects\CREXX_F030\cmake-build-debug\compiler\rxc.exe test
Error @ 14:3 - #INVALID_ASSEMBLER, "bct loop,len,offset"
1 error(s) in source file

I asked this question already, and the answer was to compile RXC. Apologies for my ignorance, how do I do that?

**Comment by rvjansen** [comment]
Hi Peter,

I guess:

make clean && make

René.

**Reply by rvjansen** [reply]
and to be less terse. yesterday I merged develop into f0030, to have function calls in rexx *and* bct - so if you git pull f0030, then do a cmake for good luck in the target directory, a make clean && make should do the trick of picking up the instruction. Note that I did a small change in f0030 machine/CMakeList.txt to use any rexx, not just regina.

**Comment by adesutherland** [comment]
Hi Peter - was this all sorted?

**Comment by Peter-Jacob** [comment]
unfortunately not, but it doesn't have any urgency. I downloaded f0030 in a new clion project, and all the other mismatches were gone! This is a relief.

**Reply by adesutherland** [reply]
Weird! CMAKE "should" have been rebuilding rxc for you automatically when the instruction was added. Lets see if you get the same problem again, perhaps it was just some one off corruption somewhere ...
